upgrade zone.js

Currently, `npm install` gives the following:
```
└── UNMET PEER DEPENDENCY zone.js@0.5.10
npm ERR! peer dep missing: zone.js@0.5.11, required by angular2@2.0.0-beta.3
```
Need to upgrade zone.js to 0.5.11 to satisfy angular 2 beta.3+ requirements.

--- a/package.json
+++ b/package.json
@@ -44,7 +44,7 @@
     "json-formatter-js": "^0.3.0",
     "reflect-metadata": "0.1.2",
     "rxjs": "5.0.0-beta.0",
-    "zone.js": "0.5.10"
+    "zone.js": "0.5.11"
   },
   "devDependencies": {
     "postcss-cssnext": "^2.4.0",

--- a/webpack.test.config.js
+++ b/webpack.test.config.js
@@ -5,8 +5,8 @@ module.exports = {
   entry: {
     'test': [
       'rxjs',
-      'zone.js/lib/browser/zone-microtask',
-      'zone.js/lib/zones/long-stack-trace',
+      'zone.js/dist/zone-microtask',
+      'zone.js/dist/long-stack-trace-zone',
       'reflect-metadata',
       path.join(__dirname, 'webpack.test.bootstrap.ts')
     ]

--- a/webpack.vendor.ts
+++ b/webpack.vendor.ts
@@ -2,8 +2,8 @@
 import 'es6-shim';
 import 'es6-promise';
 import 'reflect-metadata';
-import 'zone.js/lib/browser/zone-microtask';
-import 'zone.js/lib/zones/long-stack-trace'; // remove this for prod
+import 'zone.js/dist/zone-microtask';
+import 'zone.js/dist/long-stack-trace-zone'; // remove this for prod
 
 // Angular 2
 import 'angular2/platform/browser';

